src/v/kafka/server/handlers/create_topics.cc

```diff
@@ -239,6 +241,12 @@ ss::future<response_ptr> create_topics_handler::handle(
     // Print log if not supported configuration options are present
     for (auto& r : boost::make_iterator_range(begin, valid_range_end)) {
         for (auto c : r.configs) {
+            // special case for vcluster topic property
+            if (
+              config::shard_local_cfg().enable_mpx_extensions()
```

Comment:
is the idea here to not allow the property to be set unless mpx extensions are set? the alternative is to allow it to be set, but it doesn't do anything unless mpx extensions are enabled. well, maybe we want both? it's nice to avoid a proliferations of query configuration options.